gdk/wizard/Prompter.py

```diff
@@ -0,0 +1,259 @@
+from gdk.wizard.WizardData import WizardData
+from gdk.wizard.ConfigEnum import ConfigEnum
+from gdk.wizard.WizardChecker import WizardChecker
+from gdk.wizard.WizardConfigUtils import WizardConfigUtils
+from PyInquirer import prompt
+import argparse
+import sys
+
+
+class Wizard:
+    """
+    A class used to represent the GDK Startup Wizard
+    """
+
+    def __init__(self) -> None:
+        """
+        Initialize the Wizard object
+
+        Attributes
+        ----------
+            data: Wizard_data object
+            model: Wizard getter and setter object
+            checker: Wizard_checker object
+            parser: argparse object
+        """
+
+        self.utils = WizardConfigUtils()
+        self.project_config_file = self.utils.get_project_config_file()
+        self.field_dict = self.utils.read_from_config_file()
+
+        self.data = WizardData(self.field_dict)
+        self.checker = WizardChecker()
+        self.parser = argparse.ArgumentParser()
+
+    def prompter(self, field, required, max_attempts=3):
```

Comment:
This is for later - should we also pass help text as an arg to this method. 

For eg. when user fails to provide a valid input for `version`, the help log should be something like 
> Please input a valid version for the component in semver format. Eg. 1.0.0

Reply:
I see yes definitely. I'll put it in the TODOs list